.github/workflows/mutation.yml

```diff
@@ -0,0 +1,32 @@
+name: Mutation Testing
+
+on: [pull_request]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    name: Mutation Testing Code Review Annotations
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
```

Comment:
Can we and should we test multiple PHP versions?

Reply:
There's no benefit to testing in any particular PHP version, much less multiple versions. Mutators act on very low-level language features that aren't going to be version-dependent (if you remove a method call or flip a return value, the effect will be the same regardless of PHP version)